action-allowedlist/approved.json

```diff
@@ -182,8 +182,8 @@
     },
     {
         "actionLink": "dorny/test-reporter",
-        "actionVersion": "0d00bb14cb0cc2c9b8985df6e81dd333188224e1",
-        "tag": "1.5.0"
+        "actionVersion": "c9b3d0e2bd2a4e96aaf424dbaa31c46b42318226",
+        "tag": "1.6.0"
```

Comment:
This would require to update all workflows that are using this action, correct? Could we add the new version leaving the old one and suggest all teams to start updating and then remove the old version? To avoid having breaking changes for all repos.

Reply:
Yes, it will cause failures. I suppose that is rude of me 🤣.  Let me discuss with product owners before we merge.